nav2_costmap_2d/CMakeLists.txt

```diff
@@ -105,6 +106,19 @@ target_link_libraries(nav2_costmap_2d_client
   nav2_costmap_2d_core
 )
 
+add_library(nav2_costmap_2d_standalone SHARED
```

Comment:
This name must change, its not descriptive of what's in it

Reply:
I have very poor naming skills

Reply:
`nav2_costmap_2d_collision_checker` or something